miniconda/debian/Dockerfile

```diff
@@ -1,32 +1,20 @@
-FROM debian:latest
-
-#  $ docker build . -t continuumio/miniconda:latest -t continuumio/miniconda:4.6.14 -t continuumio/miniconda2:latest -t continuumio/miniconda2:4.6.14
-#  $ docker run --rm -it continuumio/miniconda2:latest /bin/bash
-#  $ docker push continuumio/miniconda:latest
-#  $ docker push continuumio/miniconda:4.6.14
-#  $ docker push continuumio/miniconda2:latest
-#  $ docker push continuumio/miniconda2:4.6.14
+FROM debian:buster-slim
 
 ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
-ENV PATH /opt/conda/bin:$PATH
 
-RUN apt-get update --fix-missing && apt-get install -y wget bzip2 ca-certificates \
-    libglib2.0-0 libxext6 libsm6 libxrender1 \
-    git mercurial subversion
+RUN apt-get update --fix-missing && \
+    apt-get install -y wget bzip2 ca-certificates libglib2.0-0 libxext6 libsm6 libxrender1 git mercurial subversion && \
```

Comment:
Git and friends are not small applications, and aren't often needed in deployments. Considering these can be conda installed, should all these packages be part of a base conda image? FWIW this is the base conda image I've been using for things: https://github.com/jcrist/alpine-dask-docker/blob/master/alpine-conda/Dockerfile. It's just `conda` and `tini` for init, nothing else. All other things can be added by subsequent layers.

If these development tools are needed (perhaps for backwards compat), they could maybe be added at a higher layer, and a new image ("conda-base" or something) that contains only conda could be created.

Reply:
A lot of users seem to use these images for CI so I think these tools need to be here for backwards compatibility. I like the idea of having a conda-base image but I also think that having an extra layer of images will add a perhaps unneeded level of complexity to the build/release process for these images. 
Building without git, etc. beings the image down to 265MB. Do you see any other use cases for a "conda-base" debian based image? For people who want a more lean conda docker image without any extra frills there is also a conda alpine image: https://github.com/ContinuumIO/docker-images/blob/master/miniconda/alpine/Dockerfile

Reply:
The benefit of having a standard conda base image that users can depend on is that the layers in the conda base image are shared, minimizing total disk space. The downside of maintaining such an image is you'd need to appease everyone, which is always tricky. If the main users of the conda image are CI/development cases, then what you have now seems fine.